### PR TITLE
Updated kata and kata_migration tutorials.

### PR DESCRIPTION
**Pull request by MCamp859**
Updated info for selecting runtime and edits for text flow and clarity.

Signed-off-by: MCamp859 <mary.camp@ptiglobal.net>

### CODE REVIEW

**Comment by MCamp859** [comment]
@joselamego @Dnplas @devimc  FYI, This PR includes changes requested by @mythi in [PR 186](https://github.com/clearlinux/clear-linux-documentation/pull/186).

File: source/clear-linux/tutorials/kata.rst [hunk 1]
```diff
@@ -28,24 +28,23 @@ Install Kata Containers
...
 
 .. code-block:: bash
 
    sudo mkdir -p /etc/systemd/system/docker.service.d/
-   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/51-runtime.conf
    [Service]
-   ExecStart=
-   ExecStart=/usr/bin/dockerd -D --add-runtime kata-runtime=/usr/bin/kata-runtime --default-runtime=kata-runtime
+   Environment="DOCKER_DEFAULT_RUNTIME=--default-runtime kata-runtime"
```
Comment (devimc) [inline source/clear-linux/tutorials/kata.rst]: JFYI, configuring `/etc/docker/daemon.json` is another option to enable Kata Container as the default runtime 
https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file

```json
{
    "default-runtime": "kata-runtime",
    "runtimes": {
        "kata-runtime": {
            "path": "/usr/bin/kata-runtime"
        }
    }
}
```

Comment (joselamego) [reply source/clear-linux/tutorials/kata.rst]: @MCamp859: The kata_migration tutorial is intended for users that are already using Clear Containers. These are planned to be deprecated in the following weeks so this tutorial should be also (or at least be archived). Only the kata tutorial will be kept alive in the future, so no need to merge them.

**Comment by intelkevinputnam** [comment]
LGTM

File: source/clear-linux/tutorials/kata.rst [hunk 2]
```diff
@@ -28,24 +28,23 @@ Install Kata Containers
...
 
-Configure Docker\* to use Kata Containers by default
+Configure Docker\* to use Kata Containers by default.
 
 .. code-block:: bash
 
    sudo mkdir -p /etc/systemd/system/docker.service.d/
-   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/kata-containers.conf
+   cat <<EOF | sudo tee /etc/systemd/system/docker.service.d/51-runtime.conf
    [Service]
```
Comment (mvincerx) [inline source/clear-linux/tutorials/kata.rst]: @joselamego, @devimc 
Can you please confirm that the cat command on Line 42 is correct? It ends in "...51-runtime.conf"

File: source/clear-linux/tutorials/kata.rst (hunk 1 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/kata.rst]: @devimc 
Thanks for presenting an alternative. I need clarity on your recommendations, please as to whether you recommend this alternative be added, or not. 

@joselamego 
We need to provide the best option for our users to migrate. Can you please provide your opinion if Mary's contribution is acceptable? It's not at all clear to me that from this previous dialogue both you and devimc agreed. I just need confirmation. Thank you.  

I agree that we don't need to give alternatives, given the potentially short life-span of this document. Nevertheless, we should provide current CL users with the simplest and best instructions on how to migrate.

File: source/clear-linux/tutorials/kata.rst (hunk 2 above)
Comment (joselamego) [reply source/clear-linux/tutorials/kata.rst]: LGTM

File: source/clear-linux/tutorials/kata.rst (hunk 1 above)
Comment (joselamego) [reply source/clear-linux/tutorials/kata.rst]: @mvincerx 
My comment was related to the suggestion of combining both tutorials, which I believe is not necessary. Regarding the alternative to configure the runtime, I suggest to add it as a note, leaving the current instruction as default.

Comment (mvincerx) [reply source/clear-linux/tutorials/kata.rst]: Excellent. Thanks for clarifying.  @MCamp859 can you please follow up and add this as a note?

File: source/clear-linux/tutorials/kata.rst (hunk 2 above)
Comment (mvincerx) [reply source/clear-linux/tutorials/kata.rst]: Thank you.

Comment (mvincerx) [reply source/clear-linux/tutorials/kata.rst]: @mythi.  Thank you for testing and confirming.

File: source/clear-linux/tutorials/kata.rst (hunk 1 above)
Comment (mythi) [reply source/clear-linux/tutorials/kata.rst]: What's the alternative? The .json file? I don't think that's necessary.

**Comment by mvincerx** [comment]
This PR is ready is almost ready to be merged. @MCamp859--When time permits, please add the alternative method, modifying the ` /etc/docker/daemon.json`--as discussed above.

**Comment by mythi** [comment]
```daemon.json``` does not belong in this doc. There's a mechanism already in place (```clearlinux.conf```) that gives the alternative runtimes and defines how to choose one. This doc just explains how to select kata-runtime using that.

**Comment by mvincerx** [comment]
@mythi. Thanks for providing your opinion in this dialogue. 
@MCamp859 Please disregard my previous request to add a note. 
Thanks all for this healthy dialogue and for sharing your expertise.

**Pull request merged**